VideoLocker/src/main/java/org/edx/mobile/task/GetCommentListTask.java

```diff
@@ -10,17 +10,25 @@
 
     String threadId;
     int page = 1;
+    boolean isQuestionType;
+    boolean shouldGetEndorsed;
 
-    public GetCommentListTask(Context context, String threadId, int page) {
+    public GetCommentListTask(Context context, String threadId, int page, boolean isQuestionType,
```

Comment:
I guess that's not suitable for your implementation, as it uses one
anonymous class to handle queries for both questions and threads.

On Wed, Dec 16, 2015 at 1:59 PM, Mi@N Kh@LiD <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/task/GetCommentListTask.java
> <https://github.com/edx/edx-app-android/pull/475#discussion_r47750280>:
>
> >
> > -    public GetCommentListTask(Context context, String threadId, int page) {
> > +    public GetCommentListTask(Context context, String threadId, int page, boolean isQuestionType,
>
> Having 2 constructors seems to be a better option.
>
> Not a fan of sending unnecessary objects/data to a class.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-app-android/pull/475/files#r47750280>.
>